decidim-admin/app/commands/decidim/admin/destroy_participatory_space_private_user.rb

```diff
@@ -13,6 +13,41 @@ def extra_params
           }
         }
       end
+
+      def run_after_hooks
```

Comment:
TBH, reading the implementation, i believe that has potential of reaching a very big number of queries / operations. Maybe is worth having this as a background job instead?

Reply:
Hello @alecslupu, thanks for your suggestions ! 
I am just coming back from holidays, I will look at your improvments  this week and come back to you soon. Thanks again

Reply:
@Stef-Rousset Do you have any updates on this PR ?

Reply:
Hello @alecslupu, 

The code is ready, but with the holidays, I missed colleagues to review it, sorry about that...
It should be done this week, because some of them are back now !

Reply:
the implementation in a background job is done